File: src/main/java/com/my/admin/app/commerce/model/repository/TurnRepository.java
package com.my.admin.app.commerce.model.repository;

import com.black.food.manager.model.entity.Turn;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface TurnRepository extends JpaRepository<Turn, Long> {
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/repository/TurnRepository.kt
package com.bk.reactive.app.commerce.admin.my.repository

import com.bk.reactive.app.commerce.admin.my.model.document.Turn
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository

@Repository
interface TurnRepository : ReactiveMongoRepository<Turn, Any> {
}